components/react/authSignerModal.test.tsx

```
@@ -0,0 +1,125 @@
+      <SignModal
+        visible={isOpen}
+        onClose={() => {
+          isOpen = false;
+        }}
+        approve={() => {
+          approved = true;
+        }}
+        reject={() => {
+          rejected = true;
+        }}
+      />,
```

ESLint

[error] Delete `,`

(prettier/prettier)